eth_tester/normalization/inbound.py

```diff
@@ -99,17 +99,17 @@ def _normalize_inbound_access_list(access_list):
 
 TRANSACTION_NORMALIZERS = {
     "type": identity,
-    "chain_id": identity,
+    "chainId": identity,
```

Comment:
I think we can probably pass a `compose` within the normalizers for dicts so that we pass all of them through the `dict_keys_to_snake_case` and avoid all of those multiple calls to that method randomly in the backend.

I believe this is what you had started to do with the outbound but they pass through the serializers first. So for outbound it makes more sense to handle it in the serializers since we manually build the objects there. For inbound, this would make sense to handle in the normalization step.

Reply:
I see that eels already has it's [own normalizer](eth_tester/backends/eels/eels_normalizers.py) that converts everything to camelCase and stores that in the backend class. I think we can leave that as is (for now).

I understand the point of converting from snake to camel in the inbound normalizer, however I thought inbound calls should expect camel case like the JSON-RPC spec has. The validators check that the keys are camel cased and then there is nothing to normalize other than the values themselves.

I am not sure about the snake case as part of the backend internals. It appears that EELS does a secondary normalization and that is what I've replicated with the calls to `dict_keys_to_snake_case` in pyevm. Does that make sense?

Reply:
Yeah I guess EELS is different and probably the standard now. I think ideally, each backend should just have their own normalizer and perhaps have a global utils bucket they can share things from but that isn't the case and we're going to change this behavior anyhow.

Yeah we can leave this alone for sure. I'll mark all this as resolved.